### DISCUSSION TITLE
Dynamical background issue @adamwathan

### DISCUSSION THREAD

**Discussion by boosting-my-adrenaline** [tailwindlabs/tailwindcss#6810] [General]
After migrating from v2 I faced with dynamical background color / border color / shadow color issue:

1. Dynamical background / border colors stoped working properly
**className:{`bg-${color}-500 `}**

<img width="451" alt="Снимок экрана 2021-12-30 в 21 54 32" src="https://user-images.githubusercontent.com/85952749/147780432-44e680e5-29c1-419a-a3fc-bd8eea28f99b.png">

2. after I add not working color manually to 
 **className: {`bg-${color}-500 bg-emerald-500`} ** or  className: {`bg-${color}-500   bg-amber-500`}

and then erase it back to previous **className:{`bg-${color}-500`}** it appears and works till I restart server

<img width="289" alt="Снимок экрана 2021-12-30 в 21 58 12" src="https://user-images.githubusercontent.com/85952749/147780697-01d3943b-3da9-4c7b-8de6-46f8f315601d.png">

macOS, Chrome,
"tailwindcss": "^3.0.8",
"postcss": "^8.4.5",
"react": "^17.0.2",
"react-dom": "^17.0.2",
"react-scripts": "5.0.0",
Node v16.11.0
Webpack 5.65.0

**Comment by adamwathan** [comment]
Hey! See docs:

https://tailwindcss.com/docs/content-configuration#dynamic-class-names

**Reply by boosting-my-adrenaline** [reply]
wow. v2 worked fine with it. thank you again Adam!
